### PR TITLE
Merge from devdiv

### PR DESCRIPTION
**Pull request by twsouthwick**
The biggest change this brings is the VS extension code.

### CODE REVIEW

File: src/ApiPort.VS2013/Analyze/ApiPortVsAnalyzer.cs [hunk 1]
```diff
@@ -0,0 +1,198 @@
+﻿using ApiPortVS.Resources;
```
Comment (conniey) [inline src/ApiPort.VS2013/Analyze/ApiPortVsAnalyzer.cs]: Need to run the CodeFormatter tool to add the MIT header to the *.cs files.

File: src/ApiPort.VS2013/ApiPort.VS2013.csproj [hunk 2]
```diff
@@ -0,0 +1,325 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (conniey) [inline src/ApiPort.VS2013/ApiPort.VS2013.csproj]: Rename the project from ApiPort.VS2013 back to ApiPortVS since does not open on VS2013 anymore.

Comment (conniey) [inline src/ApiPort.VS2013/ApiPort.VS2013.csproj]: Or would it be better done in another PR?

Comment (twsouthwick) [inline src/ApiPort.VS2013/ApiPort.VS2013.csproj]: Done

File: src/ApiPort.VS2013/Analyze/ApiPortVsAnalyzer.cs (hunk 1 above)
Comment (twsouthwick) [inline src/ApiPort.VS2013/Analyze/ApiPortVsAnalyzer.cs]: Done

File: src/ApiPort.Vsix/ApiPort.Vsix.sln [hunk 3]
```diff
@@ -0,0 +1,50 @@
+﻿
```
Comment (conniey) [inline src/ApiPort.Vsix/ApiPort.Vsix.sln]: We don't need this file anymore since it's integrated into PortabilityTools.sln

Comment (twsouthwick) [inline src/ApiPort.Vsix/ApiPort.Vsix.sln]: Sounds good. I just saw that file and was wondering what its use was... I've removed it now.
